<ui version="4.0" >
 <author>Borys Jurgiel</author>
 <class>QgsPluginInstallerPluginErrorDialogBase</class>
 <widget class="QDialog" name="QgsPluginInstallerPluginErrorDialogBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>383</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>480</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Error loading plugin</string>
  </property>
  <layout class="QGridLayout" >
   <item row="1" column="0" >
    <widget class="QLabel" name="label" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>The plugin seems to be invalid or have unfulfilled dependencies. It has been installed, but can't be loaded. If you really need this plugin, you can contact its author or &lt;a href="http://lists.osgeo.org/mailman/listinfo/qgis-user">QGIS users group&lt;/a> and try to solve the problem. If not, you can just uninstall it. Here is the error message below:</string>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
     <property name="openExternalLinks" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QTextBrowser" name="textBrowser" >
     <property name="minimumSize" >
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="focusPolicy" >
      <enum>Qt::NoFocus</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>503</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0" >
    <widget class="QLabel" name="label1" >
     <property name="frameShape" >
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Plain</enum>
     </property>
     <property name="text" >
      <string>Do you want to uninstall this plugin now? If you're unsure, probably you would like to do this.</string>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="0" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="standardButtons" >
      <set>QDialogButtonBox::No|QDialogButtonBox::NoButton|QDialogButtonBox::Yes</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>10</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>textBrowser</tabstop>
 </tabstops>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QgsPluginInstallerPluginErrorDialogBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>266</x>
     <y>428</y>
    </hint>
    <hint type="destinationlabel" >
     <x>266</x>
     <y>226</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QgsPluginInstallerPluginErrorDialogBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>266</x>
     <y>428</y>
    </hint>
    <hint type="destinationlabel" >
     <x>266</x>
     <y>226</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
